src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -7,10 +7,12 @@ public class CliSyntax {
 
     /* Prefix definitions */
     public static final Prefix PREFIX_NAME = new Prefix("-n");
-    public static final Prefix PREFIX_PHONE = new Prefix("-p");
+    public static final Prefix PREFIX_PHONE = new Prefix("-ph");
     public static final Prefix PREFIX_EMAIL = new Prefix("-e");
     public static final Prefix PREFIX_ADDRESS = new Prefix("-a");
     public static final Prefix PREFIX_TAG = new Prefix("-t");
     public static final Prefix PREFIX_PRICE_RANGE = new Prefix("-r");
     public static final Prefix PREFIX_CHARACTERISTICS = new Prefix("-c");
+
+    public static final Prefix PREFIX_PROPERTIES = new Prefix("-pr");
```

Comment:
Probably need to discuss what Prefixes to use. Phone, Price, Price Range, Properties can be confusing